client/src/components/Common/Wizard/useWizard.ts

```diff
@@ -25,6 +25,12 @@ export interface WizardStep {
      * This is useful for optional or conditional steps.
      */
     isSkippable: () => boolean;
+
+    /**
+     * Hack added by John, someone component in CSS should undo this or come up
+     * with an alternative that doesn't require this to be set.
+     */
+    hackBodyWidth?: string;
```

Comment:
Not finding that anywhere else. does it do anything ?

Reply:
Sample sheet PR may still use it but I dropped it - more likely than not it was some hack attempt that was undone a different way. When rebasing this morning there was a conflict with David's PR that he made some more competent changes to wizard width handling - I've made a note to review that PR and see hacks added here need to be dropped.